g2p/mappings/utils.py

```diff
@@ -808,8 +814,8 @@ def validate_preserve_case(self):
     def add_parent_dir(cls, value: Any, info: ValidationInfo):
         """If there is a parent directory, prepend it to all path fields."""
         if isinstance(value, (str, Path)):
-            if info.data.get("parent_dir", None):
-                value = Path(info.data["parent_dir"]) / value
+            if info.data.get("parent_dir", None):  # type: ignore[attr-defined]
+                value = Path(info.data["parent_dir"]) / value  # type: ignore[attr-defined]
```

Comment:
I tried to make mypy see this as correct, as it is valid Pydantic, but I found no way to make it stop saying that ValidationInfo has no data field, try as a might.

Reply:
hm. have we enabled the [Pydantic mypy plugin](https://docs.pydantic.dev/latest/integrations/mypy/#enabling-the-plugin)?  I thought we did but maybe we didn't yet

Reply:
hm, yes, it is enabled...

Reply:
what version of mypy are you using? I don't get an error here with 1.9.0

Reply:
Oh, good question! I have 1.8.0 in one env, I'll test to see if 1.9.0 fixes it.

Reply:
Nope, mypy 1.9.0, Python 3.8.17, I still get there error.

Reply:
Ah, OK, `pydantic>=2.4.0` fixes it. 2.3.0 has the warning.